osquery/tables/system/windows/hvci_status.cpp

```diff
@@ -0,0 +1,58 @@
+#include <algorithm>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+std::vector<std::string> vbs_methods = {"VBS_NOT_ENABLED",
+                                        "VBS_ENABLED_AND_NOT_RUNNING",
+                                        "VBS_ENABLED_AND_RUNNING"};
+
+std::vector<std::string> enforcement_methods = {
+    "OFF", "AUDIT_MODE", "ENFORCED_MODE"};
+
+QueryData genHVCIStatus(QueryContext& context) {
+  Row r;
```

Comment:
If you're suggesting that I move 13-20 to line 30, I received previous feedback move most of my code outside the loop, since I would be recreating variables in each loop iteration.

https://github.com/facebook/osquery/pull/5426#discussion_r260959516

Reply:
Actually, I think I'm gunna say my bad on this one and let's go with @fmanco's suggestion here :) Thinking through this, my concern would be if logic ever changes and we're not updating the values for every column in the row we might get stale data being set. That's my bad, sorry about that!

Reply:
No worries, I just moved them back into the loop.